pentaho/src/main/java/pt/webdetails/cda/CdaUtils.java

```diff
@@ -311,34 +325,26 @@ public StreamingOutput listParameters( @QueryParam( "path" ) String path,
     }
   }
 
-  private TableExporter useExporter( final CdaEngine engine, final String outputType,
-                                     HttpServletResponse servletResponse ) throws UnsupportedExporterException {
-    final ExportOptions exportOptions = getSimpleExportOptions( outputType, null );
-
-    return useExporter( engine, exportOptions, servletResponse );
-  }
-
-  private TableExporter useExporter( final CdaEngine engine, ExportOptions exportOptions,
-                                     HttpServletResponse servletResponse ) throws UnsupportedExporterException {
+  private void useExporter( final CdaEngine engine, ExportOptions exportOptions,
```

Comment:
Given that this function is no longer used to get the exporter, I suggest stop passing in "final CdaEngine engine" and pass instead the instance of TableExporter. Additionally, rename it to something like "setExportHeaders" or similar, so as to reflect what it is actually doing.